src/main/kotlin/org/veupathdb/lib/rabbit/jobs/QueueWrapper.kt

```diff
@@ -107,50 +90,46 @@ sealed class QueueWrapper {
    * Initializes the job dispatch queue if it is not already initialized.
    */
   protected open fun Channel.initDispatchQueue() {
-    queueDeclare(dispatchQueueName, true, false, false, emptyMap())
+    queueDeclare(
+      /* queue = */      dispatchQueueName,
+      /* durable = */    true,
+      /* exclusive = */  false,
+      /* autoDelete = */ false,
+      /* arguments = */  mapOf(
```

Comment:
Curious why the arg names are commented out here

Reply:
The API is defined in Java not Kotlin, so arg name usage doesn't work the same